src/main/java/org/gradle/profiler/studio/StudioGradleClient.java

```diff
@@ -76,13 +95,27 @@ public StudioGradleClient(GradleBuildConfiguration buildConfiguration, Invocatio
     }
 
     @Override
-    public void close() throws IOException {
-        System.out.println("* Stopping Android Studio....");
-        studioPluginConnection.send(new StudioRequest(EXIT));
-        studioProcess.waitForSuccess();
-        System.out.println("* Android Studio stopped.");
-        studioPluginServer.close();
-        studioAgentServer.close();
+    public void close() {
+        try {
+            System.out.println("* Stopping Android Studio....");
+            studioPluginConnection.send(new StudioRequest(EXIT));
+            studioProcess.waitForSuccess(STUDIO_EXIT_TIMEOUT_SECONDS, SECONDS);
+            System.out.println("* Android Studio stopped.");
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.out.println("* Android Studio did not stopped successfully, you will have to close it manually.");
```

Comment:
Typo: "did not stop", or better: "did not finish successfully".

Reply:
Changed to "did not finish successfully"